doc/HOSTING

```diff
@@ -0,0 +1,116 @@
+# introduction
+
+This document tries to explain what is required to host a
+dedicated server, provided you already have a working
+unvanquished installation and "pak" files (those who have a
+`.dpk` file extension), which are not necessarily the
+official ones.
+
+# non-goals
+It does *not* tries to explain how to compile or modify
+unvanquised, how to configure a server with modified engine,
+how to setup your system or network, and aims at the
+simplest possible configuration.
+
+# pre-requisites
+
+First of all, you need a computer than can be reached by the
+Internet. A computer on your LAN might do the job, but you
+will need to configure your router beforehand.
+You will also need an HTTP server serving on port 80. While
+this is not strictly necessary, it is highly recommended.
+Not doing this means that if your players lack one of the
+requirements, they might be unable to join at all.
+I'll skip the details on this, as this discussion is out
+of the scope for this current document.
+
+# Daemon installation
+
+Create a directory in which you will place the following
```

Comment:
For the purpose of the document, it would be helpful to give an example of a directory so you can refer to this directory in your subsequent examples.

Reply:
Good point.